### PR TITLE
[docs] Add http.url method

### PR DESCRIPTION
**Pull request by EdvinasDaugirdas**
Add documentation for `http.url` method.

### CODE REVIEW

File: src/data/markdown/docs/02 javascript api/08 k6-http/10-batch- requests -.md [hunk 1]
```diff
@@ -12,19 +12,20 @@ Batch multiple HTTP requests together, to issue them in parallel over multiple T
...
 | 3        | body (optional)   | string / object / ArrayBuffer | The body of the request if relevant. Can be set to `null` if not applicable but you want to set the last `params` argument. |
 | 4        | params (optional) | object           | [Params](/javascript-api/k6-http/params) like auth, custom headers and tags.                                                |
 
+
+### Returns
+
+| Type   | Description                                                                                                                                                                                                                   |
+| ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| object | An object containing [Response](/javascript-api/k6-http/response) objects.<br /><br />It is an array when users pass an array as `requests` and is a normal object with string keys when named requests are used (see below). |
+
```
Comment (EdvinasDaugirdas) [inline src/data/markdown/docs/02 javascript api/08 k6-http/10-batch- requests -.md:28]: Change unrelated to `http.url`, but seemed like a correct one?

Before:
Screenshot 2021-10-06 at 09 47 42

After:
Screenshot 2021-10-06 at 09 53 23

File: src/data/markdown/docs/02 javascript api/08 k6-http/10-url- url- .md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: 'url( url )'
+description: 'Creates a URL with a name tag (used in URL grouping).'
+excerpt: 'Creates a URL with a name tag (used in URL grouping).'
+---
+
+URLs that contain dynamic parts can introduce a large number of unique URLs in the metrics stream. To remedy this issue you can use `http.url` to set a consistent name tag in your requests.
```
Comment (ppcano) [inline src/data/markdown/docs/02 javascript api/08 k6-http/10-url- url- .md:7]: Suggested:

Read more on [URL grouping](/using-k6/http-requests/#url-grouping).

**Review by ppcano — COMMENTED** [review]
I suggested two minor additions.

**Comment by EdvinasDaugirdas** [comment]
> I suggested two minor additions.

Updated with your suggestions. 👍

**Review by ppcano — APPROVED** [review]

File: src/data/markdown/docs/02 javascript api/08 k6-http/10-batch- requests -.md (hunk 1 above)
Comment (inancgumus) [reply src/data/markdown/docs/02 javascript api/08 k6-http/10-batch- requests -.md:28]: **A tiny suggestion:**

_The returned_ object _contains_ Response objects.

It is an array when users pass an array as `requests` and is _an ordinary object_ with string keys when named requests are used (see below).

File: src/data/markdown/docs/02 javascript api/08 k6-http/10-batch- requests -.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ When each request is specified as an array, the order of the arguments for each
 
 | Type   | Description                                                                                                                                                                                                                   |
 | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| object | An object containing [Response](/javascript-api/k6-http/response) objects.<br /><br />It is an array when users pass an array as `requests` and is a normal object with string keys when named requests are used (see below). |
+| object | The returned object contains [Response](/javascript-api/k6-http/response) objects.<br /><br />It is an array when users pass an array as `requests` and is a ordinary object with string keys when named requests are used (see below). |
```
Comment (inancgumus) [inline src/data/markdown/docs/02 javascript api/08 k6-http/10-batch- requests -.md:27]: is **an** ordinary object

**Comment by na--** [comment]
hmm I think this might be more confusing than helpful
Screenshot 2021-10-06 at 16-10-08 url( url )

`http.url` is not a function, this won't work:
```js
export default function () {
    http.get(http.url(`https://test.k6.io`));
}
```

**Comment by inancgumus** [comment]
@na-- Does this `http.url` tagged template resolve to the following?

```golang
// lib/netext/httpext/url.go
type URL struct {
	u        *url.URL
	Name     string // http://example.com/thing/${}/
	URL      string // http://example.com/thing/1234/
	CleanURL string // URL with masked user credentials, used for output
}
```

**Comment by na--** [comment]
@inancgumus yes
@EdvinasDaugirdas :+1: I agree with you

**Pull request merged**